n4js-libs/packages/n4js-runtime/src/n4js/N4BuiltInTypes.n4js

```diff
@@ -70,7 +70,7 @@ abstract class N4NamedElement extends N4Element {
 	public name: string;
 }
 
-abstract class N4Type extends N4NamedElement {
+export project abstract class N4Type extends N4NamedElement {
```

Comment:
At the top of this file there is a comment saying:
```
// This has the following consequences:
// 1) neither "@@Global" nor "export" must be used in this file,
```
Since you are now exporting all the classes, change the comment at the top of file to:
```
// This has the following consequences:
// 1) neither "@@Global" nor "export" with an accessibility higher than "project" must be used in this file
//   (to ensure only the implementations, not the types declared below, are available to ordinary N4JS code),
```